premake/app.lua

```diff
@@ -16,11 +16,10 @@ function app( name )
 	xcodebuildresources 'src/%{prj.name}/Assets'
 
 	sysincludedirs {
-		'include',
```

Comment:
We still want the Engine includes in the sysincludedirs, this should probably be reverted?

Reply:
Right, that was one line removed too much after testing if it would work without the sysincludedirs in the app.lua.
Will Revert